webrender/build.rs

```diff
@@ -11,10 +11,10 @@ fn write_shaders(glsl_files: Vec<PathBuf>, shader_file_path: &Path) {
     let mut shader_file = File::create(shader_file_path).unwrap();
 
     write!(shader_file, "/// AUTO GENERATED BY build.rs\n\n").unwrap();
-    write!(shader_file, "use std::collections::HashMap;\n").unwrap();
+    write!(shader_file, "use internal_types::FastHashMap;\n").unwrap();
```

Comment:
This could use PHF, btw. Not sure how hot this lookup is, but given they're static keys seems like it could be better.